src/libraries/System.IO.Compression.ZipFile/src/System/IO/Compression/ZipFileExtensions.ZipArchiveEntry.Extract.Unix.cs

```diff
@@ -1,33 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System.Diagnostics;
-
-namespace System.IO.Compression
-{
-    public static partial class ZipFileExtensions
-    {
-        static partial void ExtractExternalAttributes(FileStream fs, ZipArchiveEntry entry)
-        {
-            Debug.Assert(!OperatingSystem.IsWindows());
-
-            // Only extract USR, GRP, and OTH file permissions, and ignore
-            // S_ISUID, S_ISGID, and S_ISVTX bits. This matches unzip's default behavior.
-            // It is off by default because of this comment:
-
-            // "It's possible that a file in an archive could have one of these bits set
-            // and, unknown to the person unzipping, could allow others to execute the
-            // file as the user or group. The new option -K bypasses this check."
-            const int ExtractPermissionMask = 0x1FF;
-            int permissions = (entry.ExternalAttributes >> 16) & ExtractPermissionMask;
-
-            // If the permissions weren't set at all, don't write the file's permissions,
```

Comment:
Do we want to keep this comment in the new code?

Reply:
I've replaced it with this comment `// .zip files created on Windows don't include permissions.`.

It's legal for a Unix zip file to have entries with zero permissions. We're not setting those due to this (existing) check.
To properly check we'd need to use `ZipVersionMadeByPlatform` from the `System.IO.Compression` assembly:

https://github.com/dotnet/runtime/blob/465fbeda31f8c55ea91f8d8c3d851a42876fa3ad/src/libraries/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.cs#L22

Reply:
> since the .zip could have been made using a previous version of .NET

Is also a pretty important piece of information